AirCasting/SearchAndFollow/CompleteScreen/CompleteScreenViewModel.swift

```diff
@@ -0,0 +1,32 @@
+// Created by Lunar on 22/02/2022.
+//
+
+import Foundation
+import SwiftUI
+
+class CompleteScreenViewModel: ObservableObject {
+    let session: SearchSession
```

Comment:
underlying session should be private and viewmodel should only expose ready to use stuff like text strings, UI-related structs etc.

Reply:
We need to pass this session to other views, like SessionHeaderView. How should we do this without exposing sessions from ViewModel?